src/harness/sas_interface.py

```diff
@@ -257,14 +257,13 @@ def PreloadRegistrationData(self, request):
     pass
 
   @abc.abstractmethod
-  def InjectExZone(self, request):
-    """Inject PPA or NTIA zone information into SAS under test.
+  def InjectExclusionZone(self, request):
+    """Inject exclusion zone information into SAS under test.
 
     Args:
-      request: A dictionary with a single key-value pair where the key is
-        "record" and the value is ZoneData object to be injected into 
-        SAS under test. For more information about ZoneData please see 
-        the SAS-SAS TS (WINNF-16-S-0096).
+      request: A dictionary with key-value pairs that injects the information 
```

Comment:
This does not match the testing API. Please fix.

Reply:
Updated   InjectExclusionZone definition to accept both GeoJSON zone and FrequencyRange   parameters